tests/data/nunit/nunit-reporter-bug-with-nested-type.json

```diff
@@ -16,7 +16,25 @@
       "stdout": "",
       "stderr": "",
       "data": null
+    },
+    {
+      "type": "case",
+      "testPath": [
+        { "type": "Assembly",    "name": "Launchable.NUnit.Test.dll" },
+        { "type": "TestSuite",   "name": "Launchable" },
+        { "type": "TestSuite",   "name": "NUnit" },
+        { "type": "TestSuite",   "name": "Test" },
+        { "type": "TestSuite",   "name": "InlineParameterTests" },
+        { "type": "TestCase",    "name": "StrTest(\"dot.in.it\")" }
```

Comment:
note:  `methodname="StrTest(&quot;dot.in.it&quot;)"`

Reply:
Are you asking why this is the correct assertion?

Reply:
No, just for my confirmation. Sorry if I made you confused.